app/controllers/results_submission_controller.rb

```diff
@@ -24,8 +24,16 @@ def last_duplicate_checker_job_run
   def newcomer_name_format_check
     competition = competition_from_params
 
-    name_format_issues = competition.accepted_newcomers.flat_map do |user|
-      ResultsValidators::PersonsValidator.name_validations(user.name, nil)
+    name_format_issues = competition.accepted_newcomers.filter_map do |user|
+      issues = ResultsValidators::PersonsValidator.name_validations(user.name)
+
+      if issues.present?
```

Comment:
@gregorbg Sorry I had to revert your suggestion because I'll be requiring `user` object to build this, and hence `presence` didn't work as expected.